src/modules/welcome/module.js

```diff
@@ -5,7 +5,10 @@ import { getChannel, sanitizeUserInput } from '../../util';
 
 const subscriptions = new Map();
 
-subscriptions.set('guildMemberAdd', async (client, member) => {
+subscriptions.set('guildMemberUpdate', async (client, oldMember, member) => {
+    // checks if the pending status (which is true while screening is still in progress) has changed.
+    // as it can only be set to false/null afterwards, this should work fine.
+    if (oldMember.pending === member.pending) return;
```

Comment:
Although I'm not sure if this is possible, if a user did somehow become pending again somehow, this would trigger again.

Reply:
This shouldn't be possible. as screening can only be initiated by discord it gets only initiated once unless discord throws some big errors, and then it isn't our fault anyways